config/routes.rb

```diff
@@ -1,6 +1,7 @@
 # rubocop:disable Metrics/BlockLength
 
 Rails.application.routes.draw do
+  use_doorkeeper
```

Comment:
Looks good to me that this doesn't cause any conflicts 👌

I think it theoretically could have if there were users with conflicting names already. Always a slight concern with routes but examining what this generates looks good.